Formula/ssh-agent-filter.rb

```diff
@@ -19,7 +19,22 @@ def install
   end
 
   test do
-    system "#{bin}/ssh-agent-filter", "-V"
+    (testpath/"test.sh").write <<-EOS.undent
+      mydir=$(pwd)
+      cd $(mktemp -d)
+      ssh-keygen -q -N '' -f keyA
```

Comment:
@tiwe-de if you know of a nice way to get the pre-generated keys in the Formula I would do that. But having them verbatim inside the file seems a bit overkill.